Makefile.am

```diff
@@ -56,6 +56,8 @@ check_PROGRAMS += \
 capn_test_SOURCES = \
 	tests/capn-test.cpp \
 	tests/capn-stream-test.cpp \
+	tests/example-test.cpp \
+	tests/example.capnp.c \
 	compiler/test.capnp.c \
 	compiler/schema-test.cpp \
 	compiler/schema.capnp.c
```

Comment:
need to add tests/example.capnp and tests/example.capnp.h in Makefile.am so they will be included in tarballs (make dist) - example.capnp needs to go to `EXTRA_DIST` and example.capnp.h into `noinst_HEADERS`

Other than that, looks good 👍